docs/modules/ow.md

```diff
@@ -118,10 +118,11 @@ Clears the search state so that it will start from the beginning again.
 Looks for the next device.
 
 #### Syntax
-`ow.search(pin)`
+`ow.search(pin, alarm_search)`
```

Comment:
Please make it clear that alarm_search is optional.

Reply:
Got any good examples? Or just mark as `(optional)` in Parameters section?

Reply:
Surround it with `[]`, see e.g. https://nodemcu.readthedocs.io/en/latest/modules/node/#nodeinfo.

Reply:
fixed!